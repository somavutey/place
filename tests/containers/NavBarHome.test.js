import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NavbarHome from "./../../components/containers/NavBarHome";

it("NavbarHome", () => {
  const wrapper = shallow(<NavbarHome />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
