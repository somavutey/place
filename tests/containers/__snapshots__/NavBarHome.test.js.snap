// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarHome 1`] = `
<Fragment>
  <nav
    className="makeStyles-navbar-1"
  >
    <Link
      className="makeStyles-navbarLogo-2"
      href="/"
    >
      <img
        alt="logo"
        height="110"
        src="/LogoProject.svg"
        width="110"
      />
    </Link>
    <ul
      className="makeStyles-navItems-3"
    >
      <li
        className="makeStyles-navButton-4"
      />
      return (
      <li
        className="makeStyles-navButton-4"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <Link
          href="/ArticleContent"
        >
          Explore Article and Videos
        </Link>
      </li>
      );
    </ul>
    <ButtonNavbar />
  </nav>
</Fragment>
`;
